docs/manifest_content.md

```diff
@@ -414,6 +422,18 @@ agents | copy | /WindowsAzure/Logs/Plugins/Microsoft.Azure.NetworkWatcher.Edp.Ne
 agents | copy | /WindowsAzure/Logs/Plugins/Microsoft.Azure.NetworkWatcher.Edp.NetworkWatcherAgentWind<br>ows/\*/\*.log
 agents | copy | /WindowsAzure/Logs/Plugins/Microsoft.Azure.NetworkWatcher.NetworkWatcherAgentWindows/<br>\*/\*.txt
 agents | copy | /WindowsAzure/Logs/Plugins/Microsoft.Azure.NetworkWatcher.NetworkWatcherAgentWindows/<br>\*/\*.log
+aks | copy | /Windows/System32/winevt/Logs/System.evtx
+aks | copy | /Windows/System32/winevt/Logs/Application.evtx
+aks | copy | /k/\*.log
+aks | copy | /k/\*.err
+aks | copy | /k/azure-vnet.log.\*
+aks | copy | /cni/azure-vnet.log
+aks | copy | /cni/azure-vnet-ipam.log
+aks | copy | /Windows/System32/winevt/Logs/Microsoft-Windows-Host-Network-Service-Admin.evtx
+aks | copy | /Windows/System32/winevt/Logs/Microsoft-Windows-Hyper-V-Compute-Admin.evtx
+aks | copy | /Windows/System32/winevt/Logs/Microsoft-Windows-Hyper-V-Compute-Operational.evtx
+aks | copy | /Windows/System32/winevt/Logs/Microsoft-Windows-Host-Network-Service-Operational.evtx
+aks | copy | /k/azure/CustomDataSetupScript.log

```

Comment:
CustomDataSetupScript.log and CustomDataSetupScript.ps1 are both in c:\AzureData folder. Not c:\k\AzureData folder.